### PR TITLE
[wasm] Performance sample - remove public nugets and clean up

### PR DESCRIPTION
**Pull request by ilonatommy**
Follow-up for https://github.com/dotnet/runtime/pull/104806.

- Removes "nuget-org" from perf sample
- Removes embedded xml from msbuild file
- Makes `src/mono/wasm/Wasm.Build.Tests/data/nuget*.config` the only place to edit in case we want to change sources.
- Requires updating `nuget9.config` name when the release version changes.

Tested locally.

### CODE REVIEW

File: src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj [hunk 1]
```diff
@@ -17,29 +17,22 @@
     <WasmExtraFilesToDeploy Include="frame-main.js" />
     <WasmExtraFilesToDeploy Include="style.css" />
     <Compile Remove="Console/Console.cs" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" ExcludeAssets="runtime" />
   </ItemGroup>
 
   <Target Name="RunSample" DependsOnTargets="RunSampleWithBrowserAndSimpleServer" />
+  <UsingTask TaskName="GetNugetConfigTask" AssemblyFile="$(OutputPath)Wasm.Browser.Bench.Sample.dll" />
+
+  <Target Name="SetNugetConfigContent" BeforeTargets="PrepareBlazorTemplate;BuildBlazorFrame">
```
Comment (radekdoulik) [inline src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:26]: `BeforeTargets` should contain `PrepareBrowserTemplate`, `BuildBlazorFrame` doesn't need to be here.

**Review by radekdoulik — APPROVED** [review]
Besides the dependencies it LGTM, thanks!

File: src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj (hunk 1 above)
Comment (ilonatommy) [reply src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:26]: It's `BuildBlazorFrame` -> `PrepareBlazorTemplate` -> `NugetConfigContent`.

**Comment by ilonatommy** [comment]
/ba-g CI does not test this change anyway

**Pull request merged**